### ISSUE TITLE
[BUG] Unknown XML-RPC tag 

### ISSUE THREAD

**Issue by Vault108** [redrobotsl/discord-sl-chat-relay#5]
**Describe the bug**
Throws an error when trying to run the bot. Previously the fix was to set the img string to null in GroupChat.JS 
https://github.com/redrobotsl/discord-sl-chat-relay/blob/204a7d6ef97795d2c2043fefdccc24c0edaa0384/SLevents/GroupChat.js#L33-L34
This not longer works, as it throws the following error: 
```js
[2023-07-29 23:10:50]: READY xxxxx#xxxx, ready to serve 179 users in 1 servers.
Error: Unknown XML-RPC tag 'H1'
    at Deserializer.onError (/root/discord-sl-chat-relay/node_modules/xmlrpc/lib/deserializer.js:115:20)
    at Deserializer.onClosetag (/root/discord-sl-chat-relay/node_modules/xmlrpc/lib/deserializer.js:208:14)
    at SAXStream.emit (node:events:514:28)
    at me._parser.<computed> [as onclosetag] (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:258:17)
    at emit (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:624:35)
    at emitNode (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:629:5)
    at closeTag (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:889:7)
    at SAXParser.write (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:1436:13)
    at SAXStream.write (/root/discord-sl-chat-relay/node_modules/sax/lib/sax.js:239:18)
    at IncomingMessage.ondata (node:internal/streams/readable:766:22)
```

**Expected behavior**
The relay will run

**Desktop (please complete the following information):**
 - OS: ANY

**Additional context**
Package List: 
```js
sl-discord-relay@0.0.2 /root/discord-sl-chat-relay
├── @caspertech/node-metaverse@0.5.31
├── @discordjs/builders@0.2.0
├── @sapphire/time-utilities@1.7.0
├── 16@0.0.2
├── axios-cache-interceptor@0.9.3
├── axios-retry@3.2.4
├── axios@0.26.0
├── colorette@1.4.0
├── discord.js@13.6.0
├── dotenv@10.0.0
├── eslint-config-airbnb-base@14.2.1
├── eslint-config-standard@16.0.3
├── eslint-plugin-import@2.25.4
├── eslint-plugin-node@11.1.0
├── eslint-plugin-promise@5.2.0
└── eslint@7.32.0
```
Node Version: v18.17.0
NPM Version: 9.6.7

**Comment by IAmNolanPerry** [comment]
Bug confirmed and acknowledged. Working on a fix

**Comment by IAmNolanPerry** [comment]
Issue: Unable to retrieve Profile Picture UUID as Second Life is weird. 
Resolution: 
- Need to Change the HTML Tag that we are looking for to looking for a <img> tag with class "avatar"

**Comment by IAmNolanPerry** [comment]
Fixed in Commit #3d37ca6cbe00
